pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/AdditionTransformFunction.java

```diff
@@ -48,14 +49,33 @@ public void init(@Nonnull List<TransformFunction> arguments, @Nonnull Map<String
       throw new IllegalArgumentException("At least 2 arguments are required for ADD transform function");
     }
 
-    for (TransformFunction argument : arguments) {
-      if (argument instanceof LiteralTransformFunction) {
-        _literalSum += Double.parseDouble(((LiteralTransformFunction) argument).getLiteral());
+    checkOperands(arguments);
```

Comment:
This is not purely check, maybe `processOperands()` is a better name?

Reply:
done